build_and_run_tests.sh

```diff
@@ -0,0 +1,11 @@
+set -x 
+
+#build
+bazel build --cxxopt='-std=c++14' //src/main:receive_main
+bazel build --cxxopt='-std=c++14' //src/main:send_main
+
+#bazel test --cxxopt='-std=c++14' --linkopt="-pthread" --test_output=all //tests:GoogleTest
+bazel run --cxxopt='-std=c++14' //tests:GoogleTest
+
+bazel clean
+set +x
```

Comment:
Would be nice that you keep the binaries at the end for the user to try to use your programm.

Reply:
Ok yes